app/models/cites_report_from_ws.rb

```diff
@@ -19,16 +20,16 @@ def initialize(sapi_country, epix_user, data)
 
   def save
     Sapi::Base.transaction do
-      if @aru.save
+      if @aru.save_wo_timestamps
```

Comment:
Since we have timestamps for both Sapi and Epix users, I'm passing the timestamps using the `copy_data` method, which will get those from the initialised aru.
Saving with timestamps will also updated the `created_at` and `updated_at` fields. I don't think we want that because, in this case for example, since the aru is coming from the WS, it means it is done by epix users, so we need to populate just the epix user timestamps